organizations/backends.py

```diff
@@ -11,6 +11,66 @@
 )
 
 
+class NoActiveCanAuthenticateMixin:
```

Comment:
I've read the documentation the pointers you've posted which lead me to [lms/envs/common.py](https://github.com/appsembler/edx-platform/blob/7960ef1a18657ae0cd2bd55a14d53dfa5365f577/lms/envs/common.py#L780-L784):

```py
AUTHENTICATION_BACKENDS = [
    'rules.permissions.ObjectPermissionBackend',
    'openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends.EdxRateLimitedAllowAllUsersModelBackend',
    'bridgekeeper.backends.RulePermissionBackend',
]
```

Looking at `EdxRateLimitedAllowAllUsersModelBackend` it turns out it extends `AllowAllUsersModelBackend` which does exactly what `NoActiveCanAuthenticateMixin` does.

```py
# Standard Django backend.
# see https://docs.djangoproject.com/en/3.1/ref/contrib/auth/#django.contrib.auth.backends.AllowAllUsersModelBackend
class AllowAllUsersModelBackend(ModelBackend):
    def user_can_authenticate(self, user):
        return True
```